integration/monai/monai_nvflare/client_algo_executor.py

```diff
@@ -56,12 +57,14 @@ def __init__(
 
         Args:
             client_algo_id (str): id pointing to the client_algo object
+            stats_sender_id (str, optional): id pointing to the AnalyticsSender object
```

Comment:
For MONAI integration, this should be `id pointing to the `LogWriterForMetricExchanger`?